crates/evm/core/src/utils.rs

```diff
@@ -103,6 +107,7 @@ pub fn new_evm_with_inspector<'a, DB, I>(
     db: DB,
     env: revm::primitives::EnvWithHandlerCfg,
     inspector: I,
+    precompiles: Vec<(Address, Precompile)>,
```

Comment:
I wanna hold off on changing this and instead limit this PR to anvil only and instead append a new handle register for new precompiles to `evm.handler.` directly in anvil

Reply:
Ah, that makes sense 👍